airflow/settings.py

```diff
@@ -26,13 +26,13 @@
 import logging
 import os
 import pendulum
-
+import sys
 
 from sqlalchemy import create_engine, exc
 from sqlalchemy.orm import scoped_session, sessionmaker
 from sqlalchemy.pool import NullPool
 
-from airflow import configuration as conf
+from airflow.configuration import conf, AIRFLOW_HOME, WEBSERVER_CONFIG  # NOQA F401
```

Comment:
If `WEBSERVER_CONFIG` is not used here, why do we have to import it? Possibly I missed something?

Reply:
It's so we can do `from airflow.settings import WEVSERVER_CONFIG`